client/src/components/Common/BookingForm/BookingForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import Calender from '../../../Images/calendar_icon.png'
+import Clock from '../../../Images/clock_icon.png'
+import RoomIcon from '../../../Images/room_icon.png'
+import './bookingForm.css'
+import axios from 'axios'
+
+class BookingForm extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      userId : 5,
+      date: '2020-06-26T07:00:00.000Z',
+      time: '9:00_11:00 Am',
+      room:  5,
+      reminder: false
+    }
+  }
+
+  handelChange = (e) => {
+    const name = e.target.name
+    const value =e.target.value
+    this.setState({[name]: value})
+  }
+  handelChecked = ()=> {
+      this.setState({reminder : this.state.reminder===false ? true : false})
+  }
+
+   handelSubmit =(e) => {
+      e.preventDefault();
+      const formData = this.state
+      axios.post( '/api/booking', formData)
+      .then(response => console.log('response :',response))
+      .catch(err => console.log(err))
+    
+  }
+
+  render () {
+    return (
+      <div>
+        <div className="date_info_continer__div">
+          <div className="date_info__div">
+            <img src={Calender} alt='calender' />
+            <h4>{this.state.date}</h4>
+          </div>
+          <div className="date_info__div">
+            <img src={Clock} alt='clock' />
+            <h4>{this.state.time}</h4>
+          </div>
+          <div className="date_info__div">
+            <img src={RoomIcon} alt='room' />
+            <h4>{this.state.room.room_name}</h4>
+          </div>
+        </div>
+        <form className="booking_form">
+          <p>Please re-fill the following fileds to complete your booking process:</p>
```

Comment:
typos, `fileds to complete` => `fields to  complete`

Reply:
thank you for all the comments , I'll work on it.